hazelcast-spring/src/test/java/com/hazelcast/spring/DummyWanReplication.java

```diff
@@ -20,6 +20,7 @@
 import com.hazelcast.map.impl.record.Record;
 import com.hazelcast.wan.ReplicationEventObject;
 import com.hazelcast.wan.WanReplicationEndpoint;
+import com.hazelcast.wan.WanReplicationEvent;
 
 public class DummyWanReplication implements WanReplicationEndpoint {
```

Comment:
This is not for mocking, it's used by spring integration tests.
See: https://github.com/hazelcast/hazelcast/blob/master/hazelcast-spring/src/test/resources/com/hazelcast/spring/fullcacheconfig-applicationContext-hazelcast.xml#L362